example/DM-mentions/.gitignore

```diff
@@ -0,0 +1,11 @@
+# Ballerina generates this directory during the compilation of a package. 
```

Comment:
I think we don't need this file as we already hv a git-ignore at the project root.

Reply:
Okay sure, i will remove it.